packages/api/src/bindings.d.ts

```diff
@@ -88,16 +105,36 @@ export interface User {
   issuer: string
   publicAddress: string
   tokens: Record<string, string>
-  github?: unknown
+  /**
+   * This will actually be json object
+   */
+  github?: string
```

Comment:
ah the types generated from the database define github as string because its `jsonb` but the client actually return a JSON object, so this is just a workaround until i find a way to improve this.